sdk/communication/communication-identity/tests.yml

```diff
@@ -5,22 +5,15 @@ extends:
     parameters:
       PackageName: "@azure/communication-identity"
       ServiceDirectory: communication
+      UseFederatedAuth: true
       CloudConfig:
         Public:
+          ServiceConnection: azure-sdk-tests
+          SubscriptionConfigurationFilePaths:
+            - eng/common/TestResources/sub-config/AzurePublicMsft.json
           SubscriptionConfigurations:
-            - $(sub-config-azure-cloud-test-resources)
             - $(sub-config-communication-services-cloud-test-resources-common)
```

Comment:
Can we use the new secret we created just for CTE here as well and remove the other ones ?

Reply:
We made it consistent and keeping the old key vault